server/express.js

```diff
@@ -1,34 +1,76 @@
 const express = require('express')
+// const formData = require('express-form-data')
+const fetch = require('node-fetch')
+
 const app = express()
+
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }))
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json())
+
 const baseURL = 'https://www.metaweather.com/api'
-const fetch = require('node-fetch')
+
+const users = [
+  {
+    id: 1,
+    username: 'john',
+    password: '1234',
+    display_name: 'johnny',
+    is_admin: true
+  },
+  {
+    id: 2,
+    username: 'luci',
+    password: '1234',
+    display_name: 'lucian',
+    is_admin: false
+  }
+]
+
+// https://stackoverflow.com/questions/4295782/how-to-process-post-data-in-node-js
+app.post('/users', (req, res) => {
```

Comment:
this is the login API, please rename the URL to `/login`, use GET method.
When you write POST `/users` that means you are creating a new user or updating an existing user.
When you write GET `/users` that means you are getting all users